src/inmanta/server/services/notificationservice.py

```diff
@@ -64,14 +64,28 @@ async def _cleanup(self) -> None:
         await data.Notification.clean_up_notifications()
 
     async def compile_done(self, compile: data.Compile) -> None:
-        if not compile.success and compile.do_export:
-            await self.notify(
-                compile.environment,
-                title="Compilation failed",
-                message="An exporting compile has failed",
-                severity=const.NotificationSeverity.error,
-                uri=f"/api/v2/compilereport/{compile.id}",
-            )
+        if not compile.success:
+            if compile.do_export:
+                await self.notify(
```

Comment:
Shouldn't we put the `pull` error in the if-block and the error regarding a failed exporting compile in the else-block? The `pull` error is a more specific error and is a more informative error. I guess we want to give a more specific error when that's possible.